src/Telegram.Bot/Requests/Available Methods/Admin Bot/UnpinChatMessageRequest.cs

```diff
@@ -17,6 +17,12 @@ public class UnpinChatMessageRequest : RequestBase<bool>
         [JsonProperty(Required = Required.Always)]
         public ChatId ChatId { get; set; }
 
+        /// <summary>
+        /// Identifier of a message to unpin. If not specified, the most recent pinned message (by sending date) will be unpinned.
+        /// </summary>
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public int MessageId { get; set; }
```

Comment:
I Think we should pass the MessageId to the Constructor, Is that correct?

Reply:
nope, according to [docs](https://core.telegram.org/bots/api#unpinchatmessage) `message_id ` is optional parameter. we should pass to constructor only required parameters.